Library/Homebrew/cmd/cleanup.rb

```diff
@@ -26,7 +26,7 @@ def cleanup_args
              description: "Show what would be removed, but do not actually remove anything."
       switch "-s",
              description: "Scrub the cache, including downloads for even the latest versions. "\
-                          "Note downloads for any installed formulae or casks will still not be deleted. "\
+                          "Note: Downloads for any installed formulae or casks will still not be deleted. "\
```

Comment:
```suggestion
                          "Note that downloads for any installed formulae or casks will still not be deleted. "\
```

Reply:
Thanks for the review, @EricFromCanada! `update-reset`, `bump-formula-pr` and the `ENVIRONMENT` section of the man page use `Note:`. Is this format reserved for cases where the note gets its own line to stand out more?

Reply:
Correct; a sentence starting with such a prefix should be its own paragraph.